inst/www/national-dq-description.md

```diff
@@ -0,0 +1,7 @@
+The table displays the volume of contacts excluded from the analysis, along with the reason for their exclusion. Many of these relate to records with missing data, such as those where ICB, age or gender are missing and can’t be derived. Nationally, the volumes for these types of missing data items are small but this can vary by ICB. 
```

Comment:
```suggestion
The table displays the volume of contacts excluded from the analysis, along with the reason for their exclusion.

Many of these relate to records with missing data, such as those where ICB, age or gender are missing and can’t be derived. Nationally, the volumes for these types of missing data items are small but this can vary by ICB. 
```